ssl/ssl_test.cc

```diff
@@ -176,11 +176,13 @@ static const CipherTest kCipherTests[] = {
         "ECDHE-ECDSA-CHACHA20-POLY1305:"
         "ECDHE-RSA-CHACHA20-POLY1305:"
         "ECDHE-ECDSA-AES128-GCM-SHA256:"
+        "ECDHE-RSA-AES128-SHA256:"
```

Comment:
Can probably come up with a million different tests here; are we happy with the current ones+the ones added in this PR?

Reply:
I think we are fine, but I can try to come up with some meaningful new tests.